documentation/docs/practical-sessions/connecting-to-mqtt.md

```diff
@@ -4,38 +4,81 @@ title: Connecting to WIS2 over MQTT
 
-As part of the WIS2 Pilot Phase in 2023, the following Global Broker services are available:
+As part of the WIS2 Pilot Phase in 2023, two Global Brokers are available, one hosted by CMA and one hosted by Meteo-France.
+
+This is an example of the WIS2-notification message structure for a message received on the topic `origin/a/wis2/arg/sabm/data/core/weather/surface-based-observations/synop`:	
+
+```json
+{
+  "id": "fa587559-b02e-40a2-9fd5-2c141c39b130",
+  "type": "Feature",
+  "version": "v04",
+  "geometry": {
+    "type": "Point",
+    "coordinates": [
+      -56.625,
+      -64.24139,
+      208
+    ]
+  },
+  "properties": {
+    "data_id": "wis2/arg/sabm/data/core/weather/surface-based-observations/synop/WIGOS_0-20000-0-89055_20230926T190000",
+    "datetime": "2023-09-26T19:00:00Z",
+    "pubtime": "2023-09-26T18:48:51Z",
+    "integrity": {
+      "method": "sha512",
+      "value": "5a239b5d2ba7a04bd3b2fa44a73a9fb98167d0d4424d7fd19c0a83c75a7715212e2b97b5db6581bb3e1895b92232614d4cc4841ee9164baf47183c8403668dbd"
+    },
+    "wigos_station_identifier": "0-20000-0-89055"
+  },
+  "links": [
+    {
+      "rel": "canonical",
+      "type": "application/x-bufr",
+      "href": "http://w2b.smn.gov.ar/data/2023-09-26/wis/arg/sabm/data/core/weather/surface-based-observations/synop/WIGOS_0-20000-0-89055_20230926T190000.bufr4",
+      "length": 254
+    },
+    {
+      "rel": "via",
+      "type": "text/html",
+      "href": "https://oscar.wmo.int/surface/#/search/station/stationReportDetails/0-20000-0-89055"
+    }
+  ]
+}
+``` 
 
-### Météo-France
+In this practical session you will learn how to use the MQTT Explorer tool to review the topics available on this Global Broker and be able to display WIS2 notification messages.
 
-- host: globalbroker.meteo.fr
-- port: 8883
-- username: everyone
-- password: everyone
+MQTT Explorer is a helpful tool to review the topic structure for a given MQTT broker and visually work with the MQTT protocol. There exist many MQTT client and server software. 
+    
+To work with MQTT programmatically (for example, in Python), you can use MQTT client libraries such as [paho-mqtt](https://pypi.org/project/paho-mqtt/) to connect to an MQTT broker and process incoming messages.
+
+## Using MQTT Explorer to connect to the Global Broker
 
-### CMA
+One way to view messages published by this Global Broker is using the MQTT Explorer which can be downloaded from the [MQTT Explorer website](https://mqtt-explorer.com).
+
+Open MQTT Explorer and add a new connection to the Global Broker hosted by CMA using the following details:
 
 - host: gb.wis.cma.cn
 - port: 8883
```

Comment:
Do we want to comment about websockets anywhere? Port 1883 and 8883 are not always available.

Reply:
Unless we have an example where we can connect over websockets to a Global Broker, 
I do not want to get too technical on day 1 and discuss it here... this session is just meant to illustrate what the WIS2 topics and notifications look like, and to make sure they can use mqtt explorer to see their notifications on the later days when working with their wis2box.